### DISCUSSION TITLE
Lychee 5.2.0 not compatible on PHP 8.2

### DISCUSSION THREAD

**Discussion by nikitakothari06** [LycheeOrg/Lychee#2360] [Q&A] [answered]
Hi,

I tried installing Lychee 5.2.0 on PHP 8.2, Mysql 5.7 and getting the following error on accessing the installer.

**A 50X error occurred
Sorry, the page you are looking for is currently unavailable.
Please try again later.**

However the script works completely fine on PHP 8.3 Is the requirement changed ?

**Comment by ildyria** [comment]
No, it should be working fine on php 8.2 especially as the CI CD is using php 8.2 and 8.3 to check.

**Reply by ildyria** [reply]
I will check the installer.

**Comment by nikitakothari06** [comment]
Ok.

I shall wait for your reply.

**Reply by ildyria** [reply]
I tried the installer locally (with sqlite & php8.2) it worked fine. Could you tell me more of your set up ?

**Reply by ildyria** [reply]
What kind of error do you get in term of layout ?
Could you give me a screenshot ?

**Comment by nikitakothari06** [comment]
I just get this.
![image](https://github.com/LycheeOrg/Lychee/assets/100770244/176eaab6-8b39-4195-932f-73134b5b7b98)

I I'll test on another server and check if my server config has an issue.

**Reply by ildyria** [reply]
I see that you are using sub directories. That error could come from there. :/

**Reply by nikitakothari06** [reply]
I have set APP_URL=http://domain.com & APP_DIR=/manual/lychee520/public in my .env file

**Reply by ildyria** [reply]
then you should be going to public instead of xxx/manual/lychee520 :)

**Reply by ildyria** [reply]
I'll try to check what is going wrong when someone access the root dir like this. :/

**Reply by nikitakothari06** [reply, marked as answer]
Hi,

I tried on my other server and it's working well in a sub directory as well.

Apologies for the troubling.

**Reply by ildyria** [reply]
No worries. It does not hurt to double check. :)
